packages.dot

```diff
@@ -2,6 +2,7 @@ digraph packages {
 	"/" -> "/models";
 	"/" -> "/services/agents";
 	"/" -> "/services/agents/grpc";
+	"/" -> "/services/checks";
```

Comment:
Please add a new service to main_test.go

Reply:
Done